source/ListActions.js

```diff
@@ -216,7 +221,8 @@
 			{from: 'open', to: '$.drawer.open'},
 			{from: 'iconSrc', to: '$.activator.src'},
 			{from: 'icon', to: '$.activator.icon'},
-			{from: 'disabled', to: '$.activator.disabled', oneWay: false}
+			{from: 'disabled', to: '$.activator.disabled', oneWay: false},
+			{from: 'allowBackKey', to: '$.drawer.allowBackKey', oneWay: false}
```

Comment:
Hi @DavidUM, thanks for all the updates! I think because `moon.ListActions` fully handles the back key support now, we can remove this binding, as well as the `onPushBackHistory` event and the `allowBackKey` property from `moon.ListActionsDrawer`.